modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua

```diff
@@ -44,7 +44,7 @@ m.hidden = {
 
 if iw and iw.mbssid_support then
 	replace = m:field(Flag, "replace", translate("Replace wireless configuration"),
-		translate("An additional network will be created if you leave this checked."))
+		translate("An additional network will be created if you leave this unchecked."))
```

Comment:
I can go through and see about fixing the translations, changing checked to unchecked shouldn't be too hard.